bin/cleanup.sh

```diff
@@ -4,3 +4,8 @@
 for namespace in istio-system istio-control istio-control-master istio-ingress istio-telemetry; do
     kubectl delete namespace $namespace --wait --ignore-not-found
 done
+
```

Comment:
Also maybe move the removal of the webhooks ?

Reply:
You mean not to use an `ownerReference` but to delete it in `cleanup.sh`?

But isn't the `ownerReference` better? You don't have the risk of having a webhook that is not working because the sidecar injector was removed.

To make this clear: when we use the `ownerReference` and delete the namespace here in `cleanup.sh` the webhook will be removed automatically.

Reply:
Interesting, didn't know this. Sounds cool. Can we do it for Galley validator as well (in istio/istio since that's generated)?